resources/automagic_dashboards/field/GenericField.yaml

```diff
@@ -1,6 +1,6 @@
 title: A look at your [[this]]
 transient_title: Here's an overview of your [[this]]
-description: How [[this]] is distributed and more.
+description: How this field is distributed, how many distinct values it has, and how it changes over time.
```

Comment:
as a user, I don't find distinct values that important that it would be a draw for me to click on this.

Reply:
Made some tweaks to this description as well as country and datetime descriptions.